File: backend/chat/src/main/java/com/booking/chat/chat/domain/KafkaMessage.java
package com.booking.chat.chat.domain;

import java.io.Serializable;
import java.time.LocalDateTime;
import lombok.AccessLevel;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.ToString;

@Getter
@ToString
@NoArgsConstructor(access = AccessLevel.PROTECTED)
@AllArgsConstructor
@Builder
public class KafkaMessage implements Serializable {

    private String message;
    private String sender;
    private LocalDateTime sendTime;
}

File: frontend/domain/src/main/java/com/ssafy/domain/model/KafkaMessage.kt
package com.ssafy.domain.model
import com.google.gson.annotations.SerializedName
import java.time.LocalDateTime

data class KafkaMessage (
    @SerializedName("message")
    val message: String,

    @SerializedName("senderId")
    val senderId: Long , // sender -> 보내는 유저 PK값

    @SerializedName("sendTime")
    val sendTime: LocalDateTime) {
}